TCAT/Utilities/SearchTableViewHelpers.swift

```diff
@@ -143,6 +175,29 @@ extension SearchResultsTableViewController: DZNEmptyDataSetSource {
     }
 }
 
+class FetchBusStops {
```

Comment:
consider encapsulating all these functions in a class. like "SearchTableviewDataManager" b/c it seems like all these functions are associated with fetching the data from disk or manipulating the data in some way. better have under some descriptive class than global functions